test/testing-payloads.js

```diff
@@ -73,12 +78,12 @@ export const faultyThingTD = {
                 {
                     href: "http://localhost:8083/TestServient/properties/temperature",
                     contentType: "application/json",
-                    op: ["readproperty", "writeproperty"],
+                    op: ["readproperty"],

```

Comment:
Similarly, to pair up the TD produced by the actual `faultyThing` and the TD provided in `testing-payloads.js`. It can be observed that when `faultyThing` is started it produces `temperature` property with `readproperty` operation only (for http(s) connections).